lib/oauth/authenticator.js

```diff
@@ -29,13 +29,21 @@ OAuthAuthenticator.prototype.authenticate = function authenticate(req,callback){
   }else{
     throw new Error('authenticate must be called with a request object and callback function');
   }
+
   var authHeader = req && req.headers && req.headers.authorization;
 
   if (authHeader){
     if(authHeader.match(/Bearer/i)){
+      var authenticator = new JwtAuthenticator(self.application);
+
+      if (this.localValidation) {
+        authenticator.withLocalValidation();
+      }
+
       var parts = authHeader.split(' ');
```

Comment:
Would be nice to have these declarations below `var authenticator ...` instead (on line 38) :)